### ISSUE TITLE
When i have the ELS script on my server the headlights on the cars with ELS just keeps blinking like if the emergency lights was on, but they are not.

### ISSUE THREAD

**Issue by helpme1234556** [MrDaGree/ELS-FiveM#36]
**Describe the bug**
When i have the ELS script on my server the headlights on the cars with ELS just keeps blinking like if the emergency lights was on, but they are not.

**To Reproduce**

**Expected behavior**
When I turn it off it should not still blink

**Screenshots**

**ELS Information:**
Version #: 1.75
Server Version #: 
Screenshots of Config.lua, vcf.lua, vcf folder:

**Additional context**

**Comment by Zeemahh** [comment]
This is probably due to a native setting the emergency lights on. Would require it to be called at a later time to prevent this.

**Comment by harman127** [comment]
I had the same issue when I ran when ELS was in my server. The issue arose from a resource called Luxart vehicle control turning wigwags on and off. Obviously Lux has two settings, on and off where as ELS had 3 for the different light stages. Being they had the same keybind ("Q"), they both would fire on the keypress. I had to write a custom function in lux to check if the vehicle I was in was an ELS vehicle. If you have any other controllers for lights and sirens I would check there first.
